File: 2st/Riyeon/가까운 1 찾기.java
class Solution {
    public int solution(int[] arr, int idx) {
        int answer = -1;
        for(int i=0; i<arr.length; i++){
            if(idx <= i && arr[i]==1){
                answer = i;
                break;
            }
        }
        return answer;
    }
}

File: 3st/Jaeyoung/가까운 1 찾기.js
function solution(arr, idx) {
    var answer = 0;
    for( var i = 0 ; i < arr.length ; i++ ) {
        if( i >= idx && arr[i] == 1 ) {
            answer = i; break;
        } else answer = -1;
    }
    return answer;
}